File: Fibonacci/Python3/Fibonacci.py
def Fibonacci(x):
	return x if x < 2 else Fibonacci(x-2) + Fibonacci(x-1)
	
print('n = ', end='')
n = int(input())
if n<0:
	n=0
print('')
for i in range(n+1):
	print('%d : %d' % (i, Fibonacci(i)))

print('')

File: Fibonacci/Ruby/Fibonacci.rb
def Fibonacci(x)
	return x if x < 2
	return Fibonacci(x-1)+Fibonacci(x-2)
end

print("n = ")
n = gets.chomp.to_i
puts("")
if n < 0 then
	n = 0
end

for i in 0..n
	print(i," : ",Fibonacci(i),"\n")
end
puts("")